wiki/models.py

```diff
@@ -22,14 +21,16 @@ def render(self):
         """
         This renders the content.
         """
-
-        # Convert [[links]] to HTML links.
-        c = re.sub(r"\[\[(\w+)\]\]", r'<a href="/wiki/\1/">\1</a>', self.content)
-
         # Render Markdown to HTML.
-        c = markdown(c)
-
-        return c
+        # See https://python-markdown.github.io/extensions/ for info.
+        return markdown(
+            self.content,
+            extensions=[
+                "fenced_code",
+                "nl2br",
+                WikiLinkExtension(base_url="/wiki/"),
```

Comment:
Would like to add syntax highlighting here but think it's slightly more involved. Parking for now